linode/networkingip/datasource_test.go

```diff
@@ -52,6 +52,7 @@ func TestAccDataSourceNetworkingIP_basic(t *testing.T) {
 					resource.TestCheckResourceAttrSet(dataResourceName, "reserved"),
 					resource.TestCheckResourceAttr(dataResourceName, "prefix", "24"),
 					resource.TestMatchResourceAttr(dataResourceName, "rdns", regexp.MustCompile(`.ip.linodeusercontent.com$`)),
+					resource.TestCheckNoResourceAttr(resourceName, "vpc_nat_1_1"),
```

Comment:
I originally wanted to add an integration test for a VPC NAT 1:1 public IPv4 address, but it ended up being pretty large and is already covered by the new unit tests.